rpc/manager/manager.proto

```diff
@@ -186,6 +186,18 @@ message InterceptSpec {
 
   // Intercept desire no default port.
   bool no_default_port = 25;
+
+  // HTTP header filters for HTTP Intercepts. Only requests containing
+  // these headers (key=value) will be intercepted. Multiple headers use AND logic.
+  map<string,string> header_filters = 27;
+
+  // Path filter patterns for HTTP Intercepts. Supports glob patterns like "/api/v1/*".
+  // If specified, only requests matching these paths will be intercepted.
+  repeated string path_filters = 28;
+
+  // Enable HTTP-aware interception mechanism for HTTP Intercepts.
+  // When true, the mechanism will parse HTTP requests to apply header/path filters.
+  bool http_mechanism = 29;
```

Comment:
This seems redundant. Why would we ever use the http mechanism unless there are header_filters and path_filters?

Reply:
removed :)